src/components/Navigation/NavBar/index.jsx

```diff
@@ -3,18 +3,29 @@ import { useDispatch, useSelector, shallowEqual } from 'react-redux';
 import classNames from 'classnames';
 import PropTypes from 'prop-types';
 
+import { languages } from 'components/LanguageWrapper';
```

Comment:
Instead of call this export function in `LanguageWrapper` we should call instead the `getAvailaleLocales` function from `utils/`.